### DISCUSSION TITLE
Add a course to resources

### DISCUSSION THREAD

**Discussion by tdserapio** [python-discord/meta#100] [General]
I have been repeatedly suggesting this to people and I think it's a really good course, especially since it teaches the basic logical flow needed to have a good grasp of programming and coding. 

I have seen other tutorials that were not good. CS50 Harvard isn't good, all you do is learn about some random functions and have some game development in the mix. The point was not really learning to actually code but to simply take it and see how programming looks like, as a prerequisite for an actual course to be taken at Harvard.

**Comment by HassanAbouelela** [comment]
What course are you referring to? You seem to have not included it anywhere.

**Comment by tdserapio** [comment]
Oh sorry https://inst.eecs.berkeley.edu/~cs61a/fa20/

**Comment by swfarnsworth** [comment]
If we were to include this resource, I would probably only want to include the [resources page](https://inst.eecs.berkeley.edu/~cs61a/fa20/resources.html) so that people aren't having to wade through details of a course they aren't taking. However, the topics that the resources cover are a bit sprawling, and I'm thinking that they depend on a very specific combination of prerequisite knowledge.

**Comment by tdserapio** [comment]
I would just want to mention that there is a much better course after that which teaches the Data Structures and Algorithms which is probably much more applicable. 

The CS61A is to prepare the person for the rigor of CS61B where one will learn a lot
